packages/core/helper-plugin/lib/src/components/LinkButton/LinkButton.stories.mdx

```diff
@@ -0,0 +1,23 @@
+<!--- Link.stories.mdx --->
+
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import { Box } from '@strapi/design-system/Box';
+import LinkButton from './index';
+
+<Meta title="components/LinkButton" component={Link} />
+
+# LinkButton
+
+React-router-dom implementation of the [Strapi Design System Link Button](https://design-system-git-main-strapijs.vercel.app/?path=/story/design-system-components-linkbutton--base)
```

Comment:
ℹ️  @soupette I've also created the LinkButton + added the link to the DS component documentation 👍🏽

Reply:
@soupette 👋 the product growth team is waiting for this PR to be merged to QA + fix QA the dark mode before the next release

Reply:
It would be awesome if we could link the props of the BaseLink as well as we don't know these components inherits from the BaseLink.